aten/src/ATen/SparseCsrTensorUtils.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <ATen/ATen.h>
+#include <ATen/SparseCsrTensorImpl.h>
+#include <ATen/SparseTensorImpl.h>
+#include <ATen/SparseTensorUtils.h>
+
+namespace at {
+namespace sparse {
```

Comment:
would it be better to put this as `sparse_csr` to more easily distinguish it from coo utils?